### PR TITLE
Improve server configuration documentation

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed 
Added English comments in the server configuration sections to improve the documentation.

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: config.lua.dist [hunk 1]
```diff
@@ -1,132 +1,235 @@
...
-worldType = "pvp"
+-- Combat settings section
+-- In this section, the following settings are defined for the server's combat settings:
+
+-- expFromPlayersLevelRange: the level range for players to gain experience from killing other players.
+-- experienceByKillingPlayers: determines if players can gain experience by killing other players.
+-- hotkeyAimbotEnabled: determines if hotkey aimbot is enabled or not.
+-- killsToBlackSkull: the number of player kills needed before a player's skull changes color to black.
+-- killsToRedSkull: the number of player kills needed before a player's skull changes color to red.
+-- pzLocked: the time in milliseconds that a player must stay in a protection zone.
```
Comment (marmichalski) [inline config.lua.dist:9]: 🤔

File: config.lua.dist [hunk 2]
```diff
@@ -1,132 +1,235 @@
...
+-- In this section, the following settings are defined for the server's connection configuration:
+
+-- allowClones: determines if multiple players from the same IP address are allowed to connect.
+-- allowWalkthrough: determines if players can walk through each other.
+-- bindOnlyGlobalAddress: determines if the server should bind only to its global IP address.
+-- enableTwoFactorAuth: determines if two-factor authentication is enabled.
+-- gameProtocolPort: the port number used for the game protocol.
+-- ip: the IP address the server will bind to.
+-- loginProtocolPort: the port number used for the login protocol.
+-- maxPacketsPerSecond: the maximum number of packets that can be sent per second.
```
Comment (marmichalski) [inline config.lua.dist:46]: Inbound? Outbound? A lot of this descriptions does not help with anything. Some of them are even wrong ☹️

File: config.lua.dist (hunk 1 above)
Comment (ranisalt) [reply config.lua.dist:9]: Was it generated by chatgpt? :laughing:

**Review by ranisalt — COMMENTED** [review]
I think it would be best to put the comment right before the config, instead of grouping like that, it's easier to find

File: config.lua.dist [hunk 3]
```diff
@@ -1,132 +1,235 @@
...
 
--- Connection Config
--- NOTE: maxPlayers set to 0 means no limit
--- NOTE: allowWalkthrough is only applicable to players
--- NOTE: two-factor auth requires token and timestamp in session key
-ip = "127.0.0.1"
+-- Connection Config section
+-- In this section, the following settings are defined for the server's connection configuration:
+
+-- allowClones: determines if multiple players from the same IP address are allowed to connect.
```
Comment (ranisalt) [inline config.lua.dist:39]: Multiple instances of the same character*

File: config.lua.dist [hunk 4]
```diff
@@ -1,132 +1,235 @@
...
+-- allowWalkthrough: determines if players can walk through each other.
+-- bindOnlyGlobalAddress: determines if the server should bind only to its global IP address.
```
Comment (ranisalt) [inline config.lua.dist:41]: This deserves a longer explanation

Comment (omarcopires) [reply config.lua.dist:41]: Could you tell me what this is about?

File: config.lua.dist (hunk 2 above)
Comment (omarcopires) [reply config.lua.dist:46]: Could you suggest fixes instead of just saying there's something wrong so I can fix it.

File: config.lua.dist (hunk 1 above)
Comment (omarcopires) [reply config.lua.dist:9]: @ranisalt some of them yes

**Pull request closed without merging**